app/Console/Commands/ArcSkunk.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Centre;
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Command\Command as CommandAlias;
+
+class ArcSkunk extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'arc:skunk';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $teviot_centre_id = 101;
+        $teviot_btg_id = 113;
+
+        $c = Centre::where("id", $teviot_centre_id)->first();
+        $reg_associated_with_fam = $c
+            ->registrations()
+            ->withFullFamily()
+            ->get()
+            ->filter(function ($f) {
+                return in_array($f->family->centre_sequence, [4, 42, 44, 46, 39, 40, 41, 45]);
+            });
+        $bundles = $reg_associated_with_fam->first()->bundles;
```

Comment:
And because if that, you want to iterate that array of registrations and fetch the reg->bundles for each.

Then iterate the bundles and set the disbursing centre id.

Reply:
This version just explodes the first bundle of the first reg